Source/RestKit/WatsonResponse.swift

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright IBM Corporation 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+// MARK: - WatsonResponse
+
+public struct WatsonResponse<T> {
```

Comment:
I added a description for WatsonResponse.

We made an architectural decision across the SDKs to define our own response type rather than expose the types of the underlying request infrastructure.  This gives us flexibility to change the implementation without creating compatibility issues.